Feliz.UseMediaQuery/UseMediaQuery.fs

```diff
@@ -0,0 +1,66 @@
+﻿module Feliz.UseMediaQuery
+
+open System
+open Browser
+open Browser.Types
+open Feliz
+
+[<RequireQualifiedAccess>]
+type ScreenSize =
+    | Mobile
+    | MobileLandscape
+    | Tablet
+    | Desktop
+    | WideScreen
+
+type Breakpoints = {
+    MobileLandscape: int
+    Tablet: int
+    Desktop: int
+    WideScreen: int
+}
+
+let defaultBreakpoints = {
+    MobileLandscape = 576
+    Tablet = 768
+    Desktop = 1024
+    WideScreen = 1216
+}
+
+let private makeQueries breakpoints =
+    let mobileQuery = sprintf "(max-width: %ipx)" breakpoints.MobileLandscape
+    let mobileLandscapeQuery = sprintf "(max-width: %ipx)" breakpoints.Tablet
+    let tabletQuery = sprintf "(max-width: %ipx)" breakpoints.Desktop
+    let desktopQuery = sprintf "(max-width: %ipx)" breakpoints.WideScreen
+    mobileQuery, mobileLandscapeQuery, tabletQuery, desktopQuery
+
+[<AutoOpen>]
+module UseMediaQueryExtension =
+     type React with
+        static member useMediaQuery (query: string) =
```

Comment:
Could you add some simple descriptions as xml comments?

Reply:
Updated.